apps/docs/nodemon.config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "verbose": true,
+  "ext": "ts",
+  "watch": ["src", ".vitepress", "../docs-snippets"],
```

Comment:
I don't believe you need to watch `src` or `.vitepress`, as Vitepress itself already considers these.

I believe the idea here is only to watch possible sources of code snippets:
 - `../docs-snippets`
 - `../../packages`